### PR TITLE
feature component

### PR DESCRIPTION
**Pull request by ChaniceStl**

### CODE REVIEW

**Review by NikoLewis — APPROVED** [review]

File: frontend/components/main/feature.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	render(){
+		console.log('FEATURE ===>',this.props.featureProps)
+		// let feature = this.props.featureProps[0]
+		return(
+			<div>
+			Feature
+			Name: 
+			Quote: 
+			Image: 
+			{store.dispatch(getFeature(2))}
```
Comment (nmadd) [inline frontend/components/main/feature.js:15]: why is this `store.dispatch` here?

File: frontend/components/main/feature.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import store from '../../store/store.js'
+import {getFeature} from '../../actions/feature-action.js'
+
+const Feature = React.createClass({
+	render(){
+		console.log('FEATURE ===>',this.props.featureProps)
```
Comment (nmadd) [inline frontend/components/main/feature.js:7]: remove `console.log` statements before merging to master

File: frontend/components/main/feature.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+		// let feature = this.props.featureProps[0]
```
Comment (nmadd) [inline frontend/components/main/feature.js:8]: remove commented out code before merging to master

File: frontend/entryRouter.js [hunk 4]
```diff
@@ -5,20 +5,27 @@ import { render } from 'react-dom';
...
 //Build Router Component
+
 const router = (
+<Provider store={store}>
 	<Router history={browserHistory}>
-		<Route path="/" component={CultureVault}>
-		<Route path='/signup' component={SignUp}/>
-		<Route path = '/explore' component={Explore} />
-		</Route>
+		<Route path='/' component={FeatureContainer}/>
```
Comment (nmadd) [inline frontend/entryRouter.js:26]: clean up this formatting. this is important!

File: frontend/reducers/feature-reducer.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+	feature:null
+}
+
+const featureReducer = (state = _defaultState , action) => {
+switch(action.type){
+		case "FETCH_DATA" :
+			return Object.assign({}, state, {feature: action.feature.data})
+		default:
+			return state;
+	}
```
Comment (nmadd) [inline frontend/reducers/feature-reducer.js:11]: clean up formatting

File: frontend/entryRouter.js [hunk 6]
```diff
@@ -5,20 +5,27 @@ import { render } from 'react-dom';
...
+import FeatureContainer from './container/main/feature-container.js'
+
 //Import Components
-import CultureVault from './cultureVault.js';
-import SignUp from './components/main/signup.js'
+// import CultureVault from './cultureVault.js';
+// import SignUp from './components/main/signup.js'
 // import Welcome from './components/main/welcome.js';
-import Explore from './components/main/explore.js';
+// import Explore from './components/main/explore.js';
```
Comment (nmadd) [inline frontend/entryRouter.js:15]: remove commented out code

File: frontend/container/main/feature-container.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+import {connect} from 'react-redux';
+
+//import the component to have reference to what you are connecting to..
+import Feature from '../../components/main/feature.js'
+
+//mapStateToProps is just a word, that passes state and makes it available to our component. 
+const mapStateToProps = state => ({
+
+	featureProps:state.featureReducer.feature
+})
```
Comment (nmadd) [inline frontend/container/main/feature-container.js:11]: clean up formatting

Comment (ChaniceStl) [reply frontend/container/main/feature-container.js:11]: Okay will do

File: frontend/reducers/feature-reducer.js (hunk 5 above)
Comment (ChaniceStl) [reply frontend/reducers/feature-reducer.js:11]: will do

File: frontend/components/main/feature.js (hunk 1 above)
Comment (ChaniceStl) [reply frontend/components/main/feature.js:15]: i need to see what if my action was working, still working on this feature component

**Pull request merged**